common/templates/mathjax_include.html

```diff
@@ -5,7 +5,9 @@
 ## MathJax configuration for Studio lives in
 ## cms/js/require-config.js.
 
+## Avoid loading mathjax if already loaded on the page
```

Comment:
@DawoudSheraz  you are changing file permissions `100644 → 100755` please keep the original file prems.